src/javascript/core/user.js

```diff
@@ -8,6 +8,7 @@ let store;
 const defaultUserConfig = {
 	storage: 'cookie',
 	name: 'spoor-id',
+	nameOverride: 'spoor-id',
 	value: null,
 	domain: (document.URL.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i)[1].indexOf('ft.com') > -1 ? 'ft.com' : null)
```

Comment:
Shall I do that in a separate PR?

Agreed on using built-in instead of a regex.  It would also currently match on "microsoft.com"...